### PR TITLE
Initial UICollectionView integration with tracking system

### PR DESCRIPTION
**Pull request by Clstroud**
collection_focus_demo mov

### CODE REVIEW

File: EyeSpeak/AppDelegate.swift [hunk 1]
```diff
@@ -11,13 +11,15 @@ import UIKit
...
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-    var window: UIWindow?
-
+    var window: HeadGazeWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
         application.isIdleTimerDisabled = true
-        CoreDataStack.loadPersistentContainer("Eyespeak")
```
Comment (jrtb) [inline EyeSpeak/AppDelegate.swift]: 👋

File: EyeSpeak/UICollectionView+Gaze.swift [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    private func selectItem(for gaze: UIHeadGaze, with event: UIHeadGazeEvent?) {
+        let point = gaze.location(in: self)
+        guard let indexPath = indexPathForItem(at: point) else {
+            for path in indexPathsForSelectedItems ?? [] {
+                deselectItem(at: path, animated: true )
+            }
+            return
+        }
+        if !(indexPathsForSelectedItems ?? []).contains(indexPath) {
+            selectItem(at: indexPath, animated: true, scrollPosition: .init())
```
Comment (jrtb) [inline EyeSpeak/UICollectionView+Gaze.swift]: does selectItem here mean hover over? assuming this is step one towards adding touch later.

Comment (Clstroud) [reply EyeSpeak/UICollectionView+Gaze.swift]: This has been rejiggered to differentiate between highlight and selection 👍

File: EyeSpeak/AppDelegate.swift (hunk 1 above)
Comment (Clstroud) [reply EyeSpeak/AppDelegate.swift]: I don't think we have any plans for Core Data long-term, so we can safely drop it for now. Once we get the rest of everything up to a point we're comfortable with, we can drop the legacy code (and Core Data along with it)

**Pull request merged**